filter/request.go

```diff
@@ -0,0 +1,62 @@
+package filter
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httputil"
+	"net/url"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+//UpdateRequest will be called back when the request is received by the proxy.
+func (f *Filter) UpdateRequest(r *http.Request) {
+	requestLog := f.log.WithFields(logrus.Fields{"url": r.URL.String(), "action": "request", "source": r.RemoteAddr})
+	requestLog.Debug("Request")
+
+	u, _ := url.Parse(f.url)
+	r.URL.Host = u.Host
+	r.Host = u.Host
+	r.URL.Scheme = u.Scheme
+	data, err := httputil.DumpRequest(r, false)
+
```

Comment:
I would prefer the blank line before data,err := 
to make clear the err comes from this